tests/tt_eager/python_api_testing/unit_testing/misc/test_optimized_conv_v2.py

```diff
@@ -92,6 +92,7 @@ def test_optimized_conv_v2(
     fuse_relu,
     packer_l1_acc,
 ):
+    pytest.skip("TODO: Delete this test suite")
```

Comment:
delete when? what are the conditions?

Reply:
There are multiple versions of conv2d currently. All versions except the latest need to be removed. All the models or tests that use the older versions must be ported to the new one, or deleted if redundant.

Reply:
Cleanup is now in progress.